perceval/backends/core/bugzillarest.py

```diff
@@ -417,13 +443,23 @@ def call(self, resource, params):
         """
         url = self.URL % {'base': self.base_url, 'resource': resource}
 
-        if self.api_token:
+        headers = None
+        if self.api_key:
+            if self.bugzilla_version == "5.0.4.rh69":
+                # This header is needed because of the instance https://bugzilla.redhat.com does not support
+                # these parameters: Bugzilla_login, Bugzilla_password, Bugzilla_token, Bugzilla_api_key
+                # More info see https://bugzilla.redhat.com/docs/en/html/api/core/v1/general.html#authentication
+                headers = {
+                    self.HAUTHORIZATION: "Bearer " + self.api_key
+                }
```

Comment:
if the version is `5.0.4.rh69` and we add the parameter with the key it will return this error 
```
You have attempted to access the API either using an unsupported method or using one or more unsupported parameters. You must use the 'Authorization' header to authenticate to the API and you must remove all unsupported parameters from the query. The unsupported parameters are: Bugzilla_login, Bugzilla_password, Bugzilla_token, Bugzilla_api_key. See...
```
Only in this version, those parameters are not supported.

But if we add the header in other versions it will work